src/surface-components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import { Card, CardImage, CardContent } from "./Card";
+import * as React from "react";
+import { Button } from "../../inputs";
+import imageUrl from "../../assets/card/card.png";
+import PropTypes, { InferProps } from "prop-types";
+import { Meta } from "@storybook/react";
+
+export default {
+  component: Card,
+} as Meta;
+
+interface CardStoryProps {
+  fullWidth?: boolean;
+}
+
+export const Default: React.FC<InferProps<CardStoryProps>> = ({
+  fullWidth,
+}) => {
+  const handlePrimaryButtonClick = () => {
+    console.log("Primary button clicked");
+  };
+
+  const handleSecondaryButtonClick = () => {
+    console.log("Secondary button clicked");
+  };
+
+  const labels = ["Click!", "DO NOT CLICK ME!"];
```

Comment:
The label should be a short string inside the button. Remove those arrays and use more appropriate names for an open-source project. For example, use the names that are provided on the design mockup: "Button Primary" and "Button Secondary".

Reply:
I had a warning message previously that I should make it an array. The solution you proposed was my first go.